acceptance/replication_test.go

```diff
@@ -73,17 +73,16 @@ func checkRangeReplication(t *testing.T, cluster *localcluster.Cluster, d time.D
 		case <-time.After(1 * time.Second):
 		}
 
-		found, err := countRangeReplicas(client)
+		foundReplicas, err := countRangeReplicas(client)
 		if err != nil {
 			return err
 		}
 
-		fmt.Fprintf(os.Stderr, "%d ", found)
-		if found == wantedReplicas {
-			fmt.Printf("... correct number of replicas found\n")
+		log.Infof("found %d replicas", foundReplicas)
```

Comment:
Was there a problem with the previous output? I sort of liked the way it printed on a single line similar to the gossip peerings test.